pkg/metadata/inventories/config.go

```diff
@@ -0,0 +1,15 @@
+package inventories
+
+import "github.com/DataDog/datadog-agent/pkg/config"
+
+// SetConfigMetadata sets the agent metadata based on the given configuration
+func SetConfigMetadata(cfg config.Config) {
```

Comment:
I added [this](https://github.com/DataDog/datadog-agent/blob/inventory/send-top-feature-info/pkg/metadata/inventories/inventories.go#L216) in a previous PR for all the static info or the one form the configuration. It's called at the inventory `init`.

Piggybacking, the init avoid having to modify all the cmd/* binary that load inventory (like dogstatsd).

WDYT of merging both ?

Reply:
Yeah, sounds good!